cpp/example_code/elasticfilesystem/create_file_system_with_modeled_exceptions.cpp

```diff
@@ -0,0 +1,120 @@
+
+//snippet-sourcedescription:[Upgrade AWS SDK for C++ to version 1.8 to build create_file_system_with_modeled_exception.cpp. This example demonstrates how to get modeled exception from CreateFileSystem operation outcome.]
```

Comment:
You might want to consider moving these purpose descriptions deeper into the code file, closer to the function declarations. Users who visually scan down into the code might miss it here.

Reply:
You mention that this demonstrates how to get a modeled exception. But it's not immediately clear to me how the code actually does this. Perhaps add some more comments to the code to help me find it better, along with any helpful tips or similar?